processor/build.gradle

```diff
@@ -5,13 +5,13 @@ archivesBaseName = 'auto-string-formatter-processor'
 dependencies {
     compile project(':runtime')
 
-    compile 'com.github.spullara.mustache.java:compiler'
-    compile 'com.google.auto:auto-common'
-    compile 'com.google.guava:guava'
-    compile 'com.squareup:javapoet'
-    compile 'javax.inject:javax.inject'
+    compile 'com.github.spullara.mustache.java:compiler:0.9.5'
+    compile 'com.google.auto:auto-common:0.10'
+    compile 'com.google.guava:guava:27.0.1-jre'
+    compile 'com.squareup:javapoet:1.11.1'
+    compile 'javax.annotation:javax.annotation-api:1.3.2'
+    compile 'javax.inject:javax.inject:1'
```

Comment:
I have a question.
When I tested this PR on my local Docker, I needed to change here.
Otherwise the following exceptions occurred.
It seems can't resolve library version.

Do you know the reason?

```
org.gradle.internal.resolve.ModuleVersionNotFoundException: Could not find com.github.spullara.mustache.java:compiler:.
```

Reply:
Could you attach `run_tests.sh` here? I could not reproduce it without that script.

At least, `ModuleVersionNotFoundException` will occur if dependency-management-plugin does not work correctly.

Reply:
I've moved the contents to `.travis.yml`, then deleted the file in https://github.com/imasahiro/auto-string-formatter/pull/15/commits/ee5abd56946f1f2d6da935ad5b994ee8c027b15b.

here is the file.
https://github.com/imasahiro/auto-string-formatter/pull/15/commits/ee5abd56946f1f2d6da935ad5b994ee8c027b15b#diff-d83a620dfa3ec1bb27fd6c6dd172e526


I installed this library to mavenLocal to test it with test module, but the maven-publish plugin may not be work in this source repo.